manager/manager.go

```diff
@@ -295,6 +295,8 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterControlServer(m.localserver, localProxyControlAPI)
 	api.RegisterHealthServer(m.localserver, localHealthServer)
 
+	localHealthServer.SetServingStatus("ControlAPI", api.HealthCheckResponse_NOT_SERVING)
```

Comment:
Do you think we should also do this for raft, for consistency? Not sure it matters though.